src/main/java/com/google/devtools/build/lib/exec/SpawnInputExpander.java

```diff
@@ -273,17 +287,35 @@ public SortedMap<PathFragment, ActionInput> getInputMapping(
       InputMetadataProvider actionInputFileCache)
       throws IOException, ForbiddenActionInputException {
     TreeMap<PathFragment, ActionInput> inputMap = new TreeMap<>();
-    addInputs(inputMap, spawn.getInputFiles(), artifactExpander, baseDirectory);
+    addInputs(inputMap, spawn.getInputFiles(), artifactExpander, spawn.getPathMapper(),
+        baseDirectory);
     addRunfilesToInputs(
         inputMap,
         spawn.getRunfilesSupplier(),
         actionInputFileCache,
         artifactExpander,
+        spawn.getPathMapper(),
         baseDirectory);
     addFilesetManifests(spawn.getFilesetMappings(), inputMap, baseDirectory);
```

Comment:
We haven't discussed this yet, but I am open to adding support. It is just that I neither know how they work nor do I see a way to perform realistic tests with them in Bazel. Also, with our current focus on Java and Starlark actions, how much of a role do they play?

Reply:
Yes, I'm definitely not asking you to do the work :) Can you please leave a comment explaining that we don't yet map them, so that a future reader understands that this is deferred work rather than an intentional part of the design?

Reply:
I added TODOs.